waku/common/wakubridge.nim

```diff
@@ -265,24 +265,28 @@ when isMainModule:
   if conf.logLevel != LogLevel.NONE:
     setLogLevel(conf.logLevel)
 
+  ## `udpPort` is only supplied to satisfy underlying APIs but is not
+  ## actually a supported transport.
+  let udpPort = conf.tcpPort
```

Comment:
The bridge has separate TCP configurations for v1 and v2. For now we can probably just read the devp2pTcpPort (v1), though it may be worth having a `devp2pUdpPort` and a `libp2pUdpPort`?

```suggestion
  let udpPort = conf.devp2pTcpPort
```